hs_core/views/resource_rest_api.py

```diff
@@ -43,6 +43,7 @@ def resourceToResourceListItem(self, r):
         science_metadata_url = site_url + reverse('get_update_science_metadata', args=[r.short_id])
         resource_map_url = site_url + reverse('get_resource_map', args=[r.short_id])
         resource_url = site_url + r.get_absolute_url()
+        coverages = [{"type": v['type'], "value": json.loads(v['_value'])} for v in r.metadata.coverages.values()]
```

Comment:
The value property doesn't apply here and the coverage data seems to be stored as a JSON string, so unfortunately I think this is necessary.